pkg/trace/api/api_test.go

```diff
@@ -368,6 +368,42 @@ func TestReceiverDecodingError(t *testing.T) {
 	})
 }
 
+func TestReceiveEndpointSniffingRequests(t *testing.T) {
```

Comment:
```suggestion
func TestDiscoveryRequest(t *testing.T) {
```
Would this be good enough?